config/gradle/common.gradle

```diff
@@ -25,6 +25,13 @@ repositories {
     }
 }
 
+dependencies {
```

Comment:
Each of the plugins has its own `configuration` that can be used to explicitly define the artifact that is used to run the check.
The term in the bracket is a shortened  expression for `(group: <X>, name: <Y>, version:<1.0>)`.